File: services/network/test/test_utils.cc
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2018 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "services/network/test/test_utils.h"

#include "services/network/public/cpp/resource_request.h"

namespace network {

std::string GetUploadData(const network::ResourceRequest& request) {
  auto body = request.request_body;
  if (!body)
    return std::string();

  CHECK_EQ(1u, body->elements()->size());
  const auto& element = body->elements()->at(0);
  CHECK_EQ(network::DataElement::TYPE_BYTES, element.type());
  return std::string(element.bytes(), element.length());
}

}  // namespace network

File: content/test/data/web_bundle/web_bundle_browsertest/testutils.js
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2019 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

function loadScript(src) {
    var s = document.createElement('script');
    s.src = src;
    document.body.appendChild(s);
}

function success() {
    document.title = 'OK';
}

function fail(msg) {
    console.error(msg);
    document.title = 'FAIL';
}

document.title = 'Ready';
